1.3.275.0: build fails because missing `volk` detection in cmake

<details>
<summary>cmake</summary>

```console
+ /usr/bin/cmake -B x86_64-redhat-linux-gnu -D BUILD_SHARED_LIBS=ON -D CMAKE_AR=/usr/bin/gcc-ar -D CMAKE_BUILD_TYPE=RelWithDebInfo -D CMAKE_C_FLAGS_RELEASE=-DNDEBUG -D CMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -D CMAKE_Fortran_FLAGS_RELEASE=-DNDEBUG -D CMAKE_INSTALL_PREFIX=/usr -D CMAKE_NM=/usr/bin/gcc-nm -D CMAKE_RANLIB=/usr/bin/gcc-ranlib -D CMAKE_VERBOSE_MAKEFILE=ON -D INCLUDE_INSTALL_DIR=/usr/include -D LIB_INSTALL_DIR=/usr/lib64 -D LIB_SUFFIX=64 -D SHARE_INSTALL_PREFIX=/usr/share -D SYSCONF_INSTALL_DIR=/etc -S . -D GLSLANG_INSTALL_DIR=/usr
-- The C compiler identification is GNU 14.0.1
-- The CXX compiler identification is GNU 14.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- Checking for module 'wayland-client'
--   Found wayland-client, version 1.22.0
-- WAYLAND_SCANNER_EXECUTABLE = /usr/bin/wayland-scanner
-- WAYLAND_PROTOCOLS_PATH = //usr/share/wayland-protocols
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Configuring done (1.7s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_Fortran_FLAGS_RELEASE
    GLSLANG_INSTALL_DIR
    INCLUDE_INSTALL_DIR
    LIB_INSTALL_DIR
    LIB_SUFFIX
    SHARE_INSTALL_PREFIX
    SYSCONF_INSTALL_DIR
```
</details>

<details>
<summary>And build fails with</summary>

```console

-- Build files have been written to: /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu
+ /usr/bin/cmake --build x86_64-redhat-linux-gnu -j48 --verbose
Change Dir: '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'

Run Build Command(s): /usr/bin/cmake -E env VERBOSE=1 /usr/bin/gmake -f Makefile -j48
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 -B/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/CMakeFiles /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu//CMakeFiles/progress.marks
/usr/bin/gmake  -f CMakeFiles/Makefile2 all
gmake[1]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f cube/CMakeFiles/vkcube.dir/build.make cube/CMakeFiles/vkcube.dir/depend
/usr/bin/gmake  -f cube/CMakeFiles/vkcubepp.dir/build.make cube/CMakeFiles/vkcubepp.dir/depend
/usr/bin/gmake  -f cube/CMakeFiles/vkcube-wayland.dir/build.make cube/CMakeFiles/vkcube-wayland.dir/depend
/usr/bin/gmake  -f vulkaninfo/CMakeFiles/vulkaninfo.dir/build.make vulkaninfo/CMakeFiles/vulkaninfo.dir/depend
/usr/bin/gmake  -f icd/CMakeFiles/VkICD_mock_icd.dir/build.make icd/CMakeFiles/VkICD_mock_icd.dir/depend
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/CMakeFiles/vkcube.dir/DependInfo.cmake "--color="
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/CMakeFiles/vkcubepp.dir/DependInfo.cmake "--color="
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/icd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/icd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/icd/CMakeFiles/VkICD_mock_icd.dir/DependInfo.cmake "--color="
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/vulkaninfo /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/vulkaninfo/CMakeFiles/vulkaninfo.dir/DependInfo.cmake "--color="
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f cube/CMakeFiles/vkcube.dir/build.make cube/CMakeFiles/vkcube.dir/build
[ 12%] Generating xdg-decoration protocol dispatch data
[ 25%] Generating xdg-decoration protocol header
[ 25%] Generating xdg-shell protocol dispatch data
[ 25%] Generating xdg-shell protocol header
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f cube/CMakeFiles/vkcubepp.dir/build.make cube/CMakeFiles/vkcubepp.dir/build
/usr/bin/gmake  -f vulkaninfo/CMakeFiles/vulkaninfo.dir/build.make vulkaninfo/CMakeFiles/vulkaninfo.dir/build
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/wayland-scanner private-code //usr/share/wayland-protocols/unstable/xdg-decoration/xdg-decoration-unstable-v1.xml /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-decoration-code.c
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/wayland-scanner private-code //usr/share/wayland-protocols/stable/xdg-shell/xdg-shell.xml /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-shell-code.c
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/wayland-scanner client-header //usr/share/wayland-protocols/unstable/xdg-decoration/xdg-decoration-unstable-v1.xml /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-decoration-client-header.h
/usr/bin/gmake  -f icd/CMakeFiles/VkICD_mock_icd.dir/build.make icd/CMakeFiles/VkICD_mock_icd.dir/build
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/wayland-scanner client-header //usr/share/wayland-protocols/stable/xdg-shell/xdg-shell.xml /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-shell-client-header.h
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0 /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/CMakeFiles/vkcube-wayland.dir/DependInfo.cmake "--color="
[ 31%] Building C object cube/CMakeFiles/vkcube.dir/cube.c.o
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/gcc -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_XCB_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -DNDEBUG -std=c99 -fPIE -fvisibility=hidden -MD -MT cube/CMakeFiles/vkcube.dir/cube.c.o -MF CMakeFiles/vkcube.dir/cube.c.o.d -o CMakeFiles/vkcube.dir/cube.c.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.c
[ 37%] Building CXX object cube/CMakeFiles/vkcubepp.dir/cube.cpp.o
[ 43%] Building CXX object vulkaninfo/CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/g++ -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_XCB_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -fno-rtti -DNDEBUG -std=c++17 -fPIE -fvisibility=hidden -fvisibility-inlines-hidden -MD -MT cube/CMakeFiles/vkcubepp.dir/cube.cpp.o -MF CMakeFiles/vkcubepp.dir/cube.cpp.o.d -o CMakeFiles/vkcubepp.dir/cube.cpp.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.cpp
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/vulkaninfo && /usr/bin/g++ -DVK_ENABLE_BETA_EXTENSIONS -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/generated -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -fno-rtti -DNDEBUG -std=c++17 -fPIE -fvisibility=hidden -fvisibility-inlines-hidden -MD -MT vulkaninfo/CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o -MF CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o.d -o CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/vulkaninfo.cpp
[ 50%] Building CXX object icd/CMakeFiles/VkICD_mock_icd.dir/mock_icd.cpp.o
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/icd && /usr/bin/g++ -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_WAYLAND_KHX -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XCB_KHX -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_KHX -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -DVkICD_mock_icd_EXPORTS -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/icd/generated -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/icd/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -fno-rtti -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wpointer-arith -Wno-unused-function -Wno-sign-compare -MD -MT icd/CMakeFiles/VkICD_mock_icd.dir/mock_icd.cpp.o -MF CMakeFiles/VkICD_mock_icd.dir/mock_icd.cpp.o.d -o CMakeFiles/VkICD_mock_icd.dir/mock_icd.cpp.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/icd/mock_icd.cpp
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f cube/CMakeFiles/vkcube-wayland.dir/build.make cube/CMakeFiles/vkcube-wayland.dir/build
gmake[2]: Entering directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
[ 56%] Building C object cube/CMakeFiles/vkcube-wayland.dir/cube.c.o
[ 62%] Building C object cube/CMakeFiles/vkcube-wayland.dir/xdg-decoration-code.c.o
[ 68%] Building C object cube/CMakeFiles/vkcube-wayland.dir/xdg-shell-code.c.o
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/gcc -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_WAYLAND_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -DNDEBUG -std=c99 -fPIE -fvisibility=hidden -MD -MT cube/CMakeFiles/vkcube-wayland.dir/cube.c.o -MF CMakeFiles/vkcube-wayland.dir/cube.c.o.d -o CMakeFiles/vkcube-wayland.dir/cube.c.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.c
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/gcc -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_WAYLAND_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -DNDEBUG -std=c99 -fPIE -fvisibility=hidden -MD -MT cube/CMakeFiles/vkcube-wayland.dir/xdg-decoration-code.c.o -MF CMakeFiles/vkcube-wayland.dir/xdg-decoration-code.c.o.d -o CMakeFiles/vkcube-wayland.dir/xdg-decoration-code.c.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-decoration-code.c
cd /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube && /usr/bin/gcc -DVK_NO_PROTOTYPES -DVK_USE_PLATFORM_WAYLAND_KHR -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube -I/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/. -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -O2 -g -grecord-gcc-switches -pipe -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fdata-sections -ffunction-sections -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -flto=auto -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Werror=format-security -Os -DNDEBUG -std=c99 -fPIE -fvisibility=hidden -MD -MT cube/CMakeFiles/vkcube-wayland.dir/xdg-shell-code.c.o -MF CMakeFiles/vkcube-wayland.dir/xdg-shell-code.c.o.d -o CMakeFiles/vkcube-wayland.dir/xdg-shell-code.c.o -c /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu/cube/xdg-shell-code.c
In file included from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.c:54:
/usr/include/volk.h:1994:21: fatal error: volk.c: No such file or directory
 1994 | #define VOLK_SOURCE "volk.c"
      |                     ^~~~~~~~
compilation terminated.
gmake[2]: *** [cube/CMakeFiles/vkcube.dir/build.make:79: cube/CMakeFiles/vkcube.dir/cube.c.o] Error 1
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[1]: *** [CMakeFiles/Makefile2:177: cube/CMakeFiles/vkcube.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
In file included from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.c:54:
/usr/include/volk.h:1994:21: fatal error: volk.c: No such file or directory
 1994 | #define VOLK_SOURCE "volk.c"
      |                     ^~~~~~~~
compilation terminated.
gmake[2]: *** [cube/CMakeFiles/vkcube-wayland.dir/build.make:103: cube/CMakeFiles/vkcube-wayland.dir/cube.c.o] Error 1
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[1]: *** [CMakeFiles/Makefile2:229: cube/CMakeFiles/vkcube-wayland.dir/all] Error 2
In file included from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/./vulkaninfo.h:92,
                 from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/generated/vulkaninfo.hpp:28,
                 from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/vulkaninfo/vulkaninfo.cpp:34:
/usr/include/volk.h:1994:21: fatal error: volk.c: No such file or directory
 1994 | #define VOLK_SOURCE "volk.c"
      |                     ^~~~~~~~
compilation terminated.
gmake[2]: *** [vulkaninfo/CMakeFiles/vulkaninfo.dir/build.make:79: vulkaninfo/CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o] Error 1
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
gmake[1]: *** [CMakeFiles/Makefile2:255: vulkaninfo/CMakeFiles/vulkaninfo.dir/all] Error 2
In file included from /home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/cube/cube.cpp:47:
/usr/include/volk.h:1994:21: fatal error: volk.c: No such file or directory
 1994 | #define VOLK_SOURCE "volk.c"
      |                     ^~~~~~~~
compilation terminated.
gmake[2]: *** [cube/CMakeFiles/vkcubepp.dir/build.make:79: cube/CMakeFiles/vkcubepp.dir/cube.cpp.o] Error 1
gmake[2]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/Vulkan-Tools-vulkan-sdk-1.3.275.0/x86_64-redhat-linux-gnu'
```
</details>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,6 +39,11 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 option(BUILD_CUBE "Build cube" ON)
 option(BUILD_VULKANINFO "Build vulkaninfo" ON)
 option(BUILD_ICD "Build icd" ON)
+option(BUILD_TESTS "Build the tests")
+option(BUILD_WERROR "Treat compiler warnings as errors")
+# NOTE: Our custom code generation target isn't desirable for system package managers or add_subdirectory users.
+# So this target needs to be off by default to avoid obtuse build errors or patches.
+option(TOOLS_CODEGEN "Enable helper codegen target")
 
 option(ENABLE_ADDRESS_SANITIZER "Use address sanitization")
 if (ENABLE_ADDRESS_SANITIZER)
@@ -50,13 +55,14 @@ endif()
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-find_package(VulkanHeaders QUIET CONFIG)
-find_package(VulkanLoader QUIET CONFIG)
-find_package(volk QUIET CONFIG)
+find_package(VulkanHeaders REQUIRED QUIET CONFIG)
+find_package(volk REQUIRED QUIET CONFIG)
+if (APPLE OR BUILD_TESTS)
+    find_package(VulkanLoader REQUIRED QUIET CONFIG)
+endif()
 
 include(GNUInstallDirs)
 
-option(BUILD_WERROR "Treat compiler warnings as errors")
 if (BUILD_WERROR)
     add_compile_options("$<IF:$<CXX_COMPILER_ID:MSVC>,/WX,-Werror>")
 endif()
@@ -96,9 +102,7 @@ elseif(MSVC)
     add_compile_options("/w34245")
 endif()
 
-# NOTE: Our custom code generation target isn't desirable for system package managers or add_subdirectory users.
-# So this target needs to be off by default to avoid obtuse build errors or patches.
-option(TOOLS_CODEGEN "Enable helper codegen target")
+
 if (TOOLS_CODEGEN)
     find_package(Python3 REQUIRED QUIET)
     add_custom_target(tools_codegen
@@ -146,7 +150,6 @@ endif()
 
 add_subdirectory(windows-runtime-installer)
 
-option(BUILD_TESTS "Build the tests")
 if(BUILD_TESTS)
     enable_testing()
     add_subdirectory(tests)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,7 +57,7 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 find_package(VulkanHeaders QUIET REQUIRED CONFIG)
 find_package(volk REQUIRED QUIET REQUIRED CONFIG)
-if (APPLE OR BUILD_TESTS)
+if ((APPLE OR BUILD_TESTS) AND NOT ANDROID)
     find_package(VulkanLoader QUIET REQUIRED CONFIG)
 endif()
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,10 +55,10 @@ endif()
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-find_package(VulkanHeaders REQUIRED QUIET CONFIG)
-find_package(volk REQUIRED QUIET CONFIG)
+find_package(VulkanHeaders QUIET REQUIRED CONFIG)
+find_package(volk REQUIRED QUIET REQUIRED CONFIG)
 if (APPLE OR BUILD_TESTS)
-    find_package(VulkanLoader REQUIRED QUIET CONFIG)
+    find_package(VulkanLoader QUIET REQUIRED CONFIG)
 endif()
 
 include(GNUInstallDirs)

